Added ignore_deleted option

rolandoalvarado opened the pull request.

RoRroland committed:
Merged with upstream

Integrated act_as_paranoid and added ignore_deleted option

Integrated act_as_paranoid and added ignore_deleted option

Integrated act_as_paranoid and added ignore_deleted option

Integrated act_as_paranoid and added ignore_deleted option

Integrated act_as_paranoid and added ignore_deleted option

Integrated act_as_paranoid and added ignore_deleted option

Integrated act_as_paranoid and added ignore_deleted option

Integrated act_as_paranoid and added ignore_deleted option

Integrated act_as_paranoid and added ignore_deleted option

Integrated act_as_paranoid and added ignore_deleted option

Integrated act_as_paranoid and added ignore_deleted option

Updated CHANGELOG

Fixed Gem::LoadError:   Specified 'sqlite3'

Fixed spec

Removed sqlite3 from gem spec

Fixed incorrect tagging alias table

Refactor specs

Added sqlite3 to Gemfile

Fixed sqlite3 version for active_record 5.0

ref Remove 'acts_as_paranoid' as dependency and add to Gemfile

Transferred acts_as_paranoid to invidividual active record versions

Minor changes to make CI pass

Minor change

Fixed spec

RoRroland committed:
Added ignore_deleted option

Added ignore_deleted option

Integrated act_as_paranoid and added ignore_deleted option

Integrated act_as_paranoid and added ignore_deleted option

Integrated act_as_paranoid and added ignore_deleted option

Integrated act_as_paranoid and added ignore_deleted option

Integrated act_as_paranoid and added ignore_deleted option

Integrated act_as_paranoid and added ignore_deleted option

Integrated act_as_paranoid and added ignore_deleted option

Integrated act_as_paranoid and added ignore_deleted option

Integrated act_as_paranoid and added ignore_deleted option

Integrated act_as_paranoid and added ignore_deleted option

Integrated act_as_paranoid and added ignore_deleted option

Integrated act_as_paranoid and added ignore_deleted option

Integrated act_as_paranoid and added ignore_deleted option

Updated CHANGELOG

Fixed Gem::LoadError:   Specified 'sqlite3'

Fixed spec

Removed sqlite3 from gem spec

Fixed incorrect tagging alias table

Refactor specs

Added sqlite3 to Gemfile

Fixed sqlite3 version for active_record 5.0

ref Remove 'acts_as_paranoid' as dependency and add to Gemfile

Transferred acts_as_paranoid to invidividual active record versions

Minor changes to make CI pass

Minor change

Fixed spec

Removed act_as_paranoid gem

Adjusted specs

Added checking for deleted_at column and added specs

Updated specs

Fixed incorrect variable and method used

RoRroland committed:
Merged with latest master

Merged with upstream

Integrated act_as_paranoid and added ignore_deleted option

Integrated act_as_paranoid and added ignore_deleted option

Integrated act_as_paranoid and added ignore_deleted option

Integrated act_as_paranoid and added ignore_deleted option

Integrated act_as_paranoid and added ignore_deleted option

Integrated act_as_paranoid and added ignore_deleted option

Integrated act_as_paranoid and added ignore_deleted option

Integrated act_as_paranoid and added ignore_deleted option

Integrated act_as_paranoid and added ignore_deleted option

Integrated act_as_paranoid and added ignore_deleted option

Integrated act_as_paranoid and added ignore_deleted option

Updated CHANGELOG

Fixed Gem::LoadError:   Specified 'sqlite3'

Fixed spec

Removed sqlite3 from gem spec

Fixed incorrect tagging alias table

Refactor specs

Added sqlite3 to Gemfile

Fixed sqlite3 version for active_record 5.0

ref Remove 'acts_as_paranoid' as dependency and add to Gemfile

Transferred acts_as_paranoid to invidividual active record versions

Minor changes to make CI pass

Minor change

Fixed spec

Removed act_as_paranoid gem

Removed act_as_paranoid gem

Added missing methods

RoRroland committed:
Integrated act_as_paranoid and added ignore_deleted option

Integrated act_as_paranoid and added ignore_deleted option

Integrated act_as_paranoid and added ignore_deleted option

Integrated act_as_paranoid and added ignore_deleted option

Integrated act_as_paranoid and added ignore_deleted option

Integrated act_as_paranoid and added ignore_deleted option

Integrated act_as_paranoid and added ignore_deleted option

Integrated act_as_paranoid and added ignore_deleted option

Integrated act_as_paranoid and added ignore_deleted option

Integrated act_as_paranoid and added ignore_deleted option

Updated CHANGELOG

Fixed Gem::LoadError:   Specified 'sqlite3'

Fixed spec

Removed sqlite3 from gem spec

Fixed incorrect tagging alias table

Refactor specs

Added sqlite3 to Gemfile

Fixed sqlite3 version for active_record 5.0

ref Remove 'acts_as_paranoid' as dependency and add to Gemfile

Transferred acts_as_paranoid to invidividual active record versions

Minor changes to make CI pass

Minor change

Fixed spec

rolandoalvarado commented:
do you think the test will work if I remove the gem and require directive?  @gzzengwei

rolandoalvarado commented:
There 6 failed CI Build which will be fix by this MR: https://github.com/mbleigh/acts-as-taggable-on/pull/951 courtesy of @damianlegawiec

rolandoalvarado commented:
@gzzengwei in our cases we don't have specific version but as stated in Contributing.md docs test needs to be pass to be merge.

rolandoalvarado commented:
@damianlegawiec I have pick few of your changes to make CI green. I will just rebase with master once your PR is merged. Thanks in advance.

brissenden commented on gemfiles/activerecord_5.1.gemfile:
```diff
@@ -11,6 +11,7 @@ when "mysql"
 else
   gem 'sqlite3'
 end
+gem 'acts_as_paranoid'
```
Why not remove acts_as_paranoid dependency completely from Gemfiles, and instead check in code if `deleted_at` column exists or not

rolandoalvarado replied:
Okay will find a way. Thanks @brissenden

rolandoalvarado committed:
Removed act_as_paranoid gem

rolandoalvarado committed:
Fixed for Not unique table/alias: 'client_taggings_56d91d7'

Make sure alias table is unique to prevent not unique table/alias
error. This is just a quick fix. I’m open for any suggestion.

rolandoalvarado committed:
Fixed specs

rolandoalvarado committed:
removed commented code

rolandoalvarado committed:
Updated Rails version for dirty_spec

rolandoalvarado committed:
Removed not need change

rolandoalvarado committed:
Added case_sensitive: false to validates_uniqueness_of directive

rolandoalvarado committed:
Update tag_spec

rolandoalvarado committed:
Removed act_as_paranoid

rolandoalvarado committed:
Updated CHANGELOG
